src/main/kotlin/com/sourcegraph/cody/vscode/InlineAutocompleteItem.kt

```diff
@@ -1,8 +1,12 @@
 package com.sourcegraph.cody.vscode
 
 class InlineAutocompleteItem(
-    val insertText: String,
+    var insertText: String,
     val filterText: String,
-    val range: Range,
+    var range: Range,
     val command: Command
-)
+) {
+  override fun toString(): String {
+    return "InlineAutocompleteItem(insertText='$insertText', filterText='$filterText', range=$range, command=$command)"
+  }
```

Comment:
Do we need `toString`?

Reply:
If we change this to a data class, we could get it automatically. Do `Range` or `Command` prohibit this being a data class?